helper/cicheck.go

```diff
@@ -95,15 +95,12 @@ func ProcessCiCheck(fileID, riskThreshold int) {
 		)
 	}
 	vulLen := len(vulnerableAnalyses)
-	linkMsg := fmt.Sprintf("\nFor details: https://secure.appknox.com/file/%d\n", fileID)
```

Comment:
Instead of removing, why not change the message to something like this? 
```
Check appknox file ID `id` for more details
```

There should be an action for the user in the failure context. That's why this message is important.

(We'll later add an `info` API in server to get the frontend URL.)